site/content/en/docs/administration/basics/AWS-Deployment-Guide.md

```diff
@@ -22,13 +22,27 @@ There are two ways of deploying the CVAT.
    [installation instructions](/docs/administration/basics/installation/).
    The additional step is to add a [security group and rule to allow incoming connections](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-network-security.html).
 
+**minimal requirements**
+
+Tested instances : t2. Crashes were reported by `cvat_redis` due to low memory. Consult [this blog post](https://docs.keydb.dev/blog/2019/12/16/blog-post/) for more details.
+t2.2xlarge | t2.xlarge and lower
+---|---
+✅ | ❌
+
 For any of above, don't forget to set the `CVAT_HOST` environment variable to the exposed
 AWS public IP address or hostname:
 
 ```bash
 export CVAT_HOST=your-instance.amazonaws.com
 ```
 
+You can also create a `.env` file in the cvat repository. The adress is saved even when restarting the server.
```

Comment:
```suggestion
You can also create a `.env` file in the cvat repository. The address is saved even when restarting the server.
```

Reply:
I'm not sure what you want to say here.

Reply:
@nmanovic instead of having to export your variable on AWS instance restart, you can store it in a .env file. See the [docker docs](https://docs.docker.com/compose/environment-variables/set-environment-variables/).
This was useful in my case, and in my opinion it is better to setup the env once at install and not having to do it at each server restart. Maybe my sentencing is not so clear, feel free to reword.